vcf-dumper/vcf-dumper-ws/src/main/java/embl/ebi/variation/eva/vcfdump/server/VcfDumperWSConfig.java

```diff
@@ -0,0 +1,58 @@
+package embl.ebi.variation.eva.vcfdump.server;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.AsyncConfigurerSupport;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.concurrent.Executor;
+
+/**
+ * Created by pagarcia on 30/09/2016.
+ */
+@Configuration
+@EnableAsync
+@EnableScheduling
+@EnableSwagger2
+public class VcfDumperWSConfig extends AsyncConfigurerSupport {
+
+    @Override
+    @Bean
+    public Executor getAsyncExecutor() {
```

Comment:
asynx-supported in the web.xml is necessary, so I have removed the EnableAsync and EnableScheduling annotations from this class (407e77d), and the method "getAsyncExecutor" is still being called.